site/ko/beta/tutorials/eager/automatic_differentiation.ipynb

```diff
@@ -0,0 +1,351 @@
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/ko/beta/tutorials/eager/automatic_differentiation.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />구글 코랩(Colab)에서 실행하기</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/ko/beta/tutorials/eager/automatic_differentiation.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />깃허브(GitHub) 소스 보기</a>\n",
+    "  </td>\n",
+    "  <td>  \n",
+    "  <a href=\"https://storage.googleapis.com/tensorflow_docs/site/ko/beta/tutorials/eager/automatic_differentiation.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />노트북 다운로드</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: 이 문서는 텐서플로 커뮤니티에서 번역했습니다. 커뮤니티 번역 활동의 특성상 정확한 번역과 최신 내용을 반영하기 위해 노력함에도\n",
+    "불구하고 [공식 영문 문서](https://www.tensorflow.org/?hl=en)의 내용과 일치하지 않을 수 있습니다.\n",
+    "이 번역에 개선할 부분이 있다면\n",
+    "[tensorflow/docs](https://github.com/tensorflow/docs) 깃헙 저장소로 풀 리퀘스트를 보내주시기 바랍니다.\n",
+    "문서 번역이나 리뷰에 지원하려면\n",
+    "[docs-ko@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs)로\n",
+    "메일을 보내주시기 바랍니다."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "vDJ4XzMqodTy"
+   },
+   "source": [
+    "이전 튜토리얼에서는 텐서(tensor)와 텐서의 연산에 대해서 알아보았습니다. 이번 튜토리얼에서는 머신러닝 모델을 최적화할 수 있는 주요 기술 중 하나인 [자동 미분(automatic differentiation)](https://en.wikipedia.org/wiki/Automatic_differentiation)에 대해 알아보겠습니다."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "GQJysDM__Qb0"
+   },
+   "source": [
+    "## 설정\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "cxzaxo6ff2y3"
+   },
+   "outputs": [],
+   "source": [
+    "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+    "\n",
+    "!pip install tensorflow==2.0.0-beta1\n",
+    "import tensorflow as tf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "1CLWJl0QliB0"
+   },
+   "source": [
+    "## 그래디언트 테이프\n",
+    "\n",
+    "텐서플로는 자동 미분(주어진 입력 변수에 대한 연산의 그래디언트(gradient)를 계산하는 것)을 위한 [tf.GradientTape](https://www.tensorflow.org/api_docs/python/tf/GradientTape) API를 제공합니다. `tf.GradientTape`는 안에서 실행된 모든 연산을 테이프(tape)에 \"기록\"합니다. 그리고 [후진 방식 자동 미분(reverse mode differentiation)](https://en.wikipedia.org/wiki/Automatic_differentiation)을 사용한 \"기록된\" 계산의 그래디언트를 계산하기 위해 기록된 연산에 관련된 테이프와 그래디언트를 사용합니다.\n",
```

Comment:
tf.GradientTape's context is meaningful to most python programmers.
I suggest "tf.GradientTape는 안에서" --> "tf.GradientTape 컨텍스트(context) 안에서".

Most talented Korean writers advise not too much to use conjunctions.(그리고, 그래서, 하지만.. etc) But I think that you can translate 'then'.
Last sentence has duplicate context like 'TF use gradient to compute gradient'. It can be simplifed just like 'TF compute gradient'.
I suggest
'그리고 후진 방식 자동 미분(reverse mode differentiation)을 사용한 "기록된" 계산의 그래디언트를 계산하기 위해 기록된 연산에 관련된 테이프와 그래디언트를 사용합니다.'
-->
'그다음 텐서플로는 후진 방식 자동 미분(reverse mode differentiation)을 사용해 테이프에 "기록된" 연산의 그래디언트를 계산합니다.'